source/docs/software/commandbased/convenience-features.rst

```diff
@@ -98,11 +98,11 @@ The ``SelectCommand`` class (`Java <https://first.wpi.edu/wpilib/allwpilib/docs/
 
   .. group-tab:: Java
 
-    .. remoteliteralinclude:: https://raw.githubusercontent.com/wpilibsuite/allwpilib/v2020.3.2/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/selectcommand/RobotContainer.java
+    .. remoteliteralinclude:: https://raw.githubusercontent.com/wpilibsuite/allwpilib/main/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/selectcommand/RobotContainer.java
```

Comment:
Do we want this to point at main or a fixed version?

Reply:
Fixed version was old before I guess we could point it to the newest fixed version

Reply:
The reason to used fixed versions is so that an update to the example doesn't break the documentation if lines shift.

Reply:
The fix for the future is the literal checker PR